src/com/hms/flexyosisoftconnector/Main.java

```diff
@@ -15,9 +15,10 @@
  */
 public class Main {
 
-  /** Application Version Numbers */
-  static final int MAJOR_VERSION = 1;
+  /** Application Major Version Number */
+  static final int MAJOR_VERSION = 2;
 
+  /** Application Minor Version Number */

```

Comment:
one commit for changing the number, another for changing the comments.

For this commit the message is "edited Version numbers" so i would expect the values to change. But the commit message does not address the comments changing.